### PR TITLE
Purge "will" from Admin Guide

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Plus the occasional drive-by.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]
Nice work looks good!

File: modules/administration/pages/infra-maintenance.adoc [hunk 1]
```diff
@@ -106,11 +106,11 @@ endif::[]
...
 === Client Tools
 
-When the server is updated consider to update some tools on the clients, too.
+When the server is updated consider updating some tools on the clients, too.
 Updating [package]``salt-minion``, [package]``zypper``, and other related management package on clients is not a strict requirement, but it is a best practice in general.
 For example, a maintenance update on the server might introduce a major new Salt version.
-Then minions will continue to work but might experience problems later on.
-To avoid this always update the salt-minion package when available.
+Then Salt clients continue to work but might experience problems later on.
+To avoid this always update the ``salt-minion`` package when available.
```
Comment (keichwa) [inline modules/administration/pages/infra-maintenance.adoc:113]:
```suggestion
To avoid this always update the [package]``salt-minion`` package when available.
```

File: modules/administration/pages/maintenance-windows.adoc [hunk 2]
```diff
@@ -69,9 +69,9 @@ You can create [path]``ical`` files in your preferred calendaring tool, such as
 [NOTE]
 ====
 When you assign a new maintenance schedule to a client, it is possible that the client might already have some restricted actions scheduled, and that these might now conflict with the new maintenance schedule.
-If this occurs, the {webui} will display an error and you will not be able to assign the schedule to the client.
+If this occurs, the {webui} displays an error and you can not assign the schedule to the client.
```
Comment (keichwa) [inline modules/administration/pages/maintenance-windows.adoc:72]:
```suggestion
If this occurs, the {webui} displays an error and you cannot assign the schedule to the client.
```

File: modules/administration/pages/mirror-sources.adoc [hunk 3]
```diff
@@ -25,7 +25,7 @@ spacewalk-service restart
 Currently, this feature can only be enabled globally for all repositories.
 It is not possible to select individual repositories for mirroring.
 
-When this feature has been activated, the source packages will become available in the {productname} {webui}
+When this feature has been activated, the source packages become available in the {productname} {webui}
```
Comment (keichwa) [inline modules/administration/pages/mirror-sources.adoc:28]:
```suggestion
When this feature has been activated, the source packages become available in the {productname} {webui} after the next repository synchronization.
```

File: modules/administration/pages/mirror-sources.adoc [hunk 4]
```diff
@@ -25,7 +25,7 @@ spacewalk-service restart
...
 after the next repository synchronization.
```
Comment (keichwa) [inline modules/administration/pages/mirror-sources.adoc:29]:
```suggestion
```

File: modules/administration/pages/monitoring.adoc [hunk 5]
```diff
@@ -256,15 +256,15 @@ The exporter packages are pre-installed in {productname} Server and Proxy, but t
 . Restart Tomcat and Taskomatic.
 . Navigate to the URL of your Prometheus server, on port 9090 (for example, [literal]``http://example.com:9090``)
 . In the Prometheus UI, navigate to menu:[Status > Targets] and confirm that all the endpoints on the ``mgr-server`` group are up.
-. If you have also installed Grafana with the {webui}, the server insights will be visible on the {productname} Server dashboard.
+. If you have also installed Grafana with the {webui}, the server insights arevisible on the {productname} Server dashboard.
```
Comment (keichwa) [inline modules/administration/pages/monitoring.adoc:259]:
```suggestion
. If you have also installed Grafana with the {webui}, the server insights are visible on the {productname} Server dashboard.
```

File: modules/administration/pages/reports.adoc [hunk 6]
```diff
@@ -84,6 +84,6 @@ This table lists the available reports:
 |===
 
 For more information about an individual report, run [command]``spacewalk-report`` with the option [option]``--info`` or [option]``--list-fields-info`` and the report name.
-The description and list of possible fields in the report will be shown.
+This shows the description and list of possible fields in the report.
 
 For further information on program invocation and options, see the [literal]``spacewalk-report(8)`` man page as well as the [option]``--help``parameter of the [command]``spacewalk-report`` command.
```
Comment (keichwa) [inline modules/administration/pages/reports.adoc:89]:
```suggestion
For further information on program invocation and options, see the [literal]``spacewalk-report(8)`` man page as well as the [option]``--help`` parameter of the [command]``spacewalk-report`` command.
```
drive-by...

File: modules/administration/pages/ssl-certs.adoc [hunk 7]
```diff
@@ -6,7 +6,7 @@
 Every client that uses SSL to register to the {productname} Server checks that it is connecting to the right server by validating against a server certificate.
 This process is called an SSL handshake.
 
-During the SSL handshake, the client will check that the hostname in the server certificate matches what it expects.
+During the SSL handshake, the client check that the hostname in the server certificate matches what it expects.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc:9]:
```suggestion
During the SSL handshake, the client checks that the hostname in the server certificate matches what it expects.
```

File: modules/administration/pages/ssl-certs.adoc [hunk 8]
```diff
@@ -47,8 +47,8 @@ The server certificate must be defined at the ``SERVER_CERT`` environment variab
 By default, {productname} uses a self-signed certificate.
 For additional security, you can arrange a third party CA to sign your certificates.
 Third party CAs perform checks to ensure that the information contained in the certificate is correct.
-They will usually charge an annual fee for this service.
-Using a third party CA makes certificates harder to spoof, and will provide additional protection for your installation.
+They usually charge an annual fee for this service.
+Using a third party CA makes certificates harder to spoof, and provide additional protection for your installation.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc:51]:
```suggestion
Using a third party CA makes certificates harder to spoof, and provides additional protection for your installation.
```

File: modules/administration/pages/tshoot-registerclones.adoc [hunk 9]
```diff
@@ -26,14 +26,14 @@ A clone is a VM that uses a primary disk that is an exact copy of an existing di
 While cloning VMs can save you a lot of time, the duplicated identifying information on the disk can sometimes cause problems.
 
 If you have a client that is already registered, you create a clone of that client, and then try and register the clone, you probably want {productname} to register them as two separate clients.
-However, if the machine ID in both the original client and the clone is the same, {productname} will register both clients as one system, and the existing client data will be over-written with that of the clone.
+However, if the machine ID in both the original client and the clone is the same, {productname} registers both clients as one system, and the existing client data is over-written with that of the clone.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-registerclones.adoc:29]:
```suggestion
However, if the machine ID in both the original client and the clone is the same, {productname} registers both clients as one system, and the existing client data is overwritten with that of the clone.
```

**Review by keichwa — APPROVED** [review]
thanks for this heroic PR!

File: modules/administration/pages/image-management.adoc [hunk 10]
```diff
@@ -409,7 +410,7 @@ For an example of a complete Kiwi repository served by git, see https://github.c
 
 [NOTE]
 ====
-You will need at least 1{nbsp}GB of RAM available for Hosts running OS Images built with Kiwi.
+You need at least 1{nbsp}GB of RAM available for Hosts running OS Images built with Kiwi.
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc:413]:
```suggestion
You need at least 1{nbsp}GB of RAM available for hosts running OS Images built with Kiwi.
```
